backend/src/main/kotlin/com/github/quillraven/toilapp_dataloader/DataLoaderApplication.kt

```diff
@@ -0,0 +1,144 @@
+import com.github.quillraven.toilapp.model.Toilet
+import com.github.quillraven.toilapp.model.User
+import com.github.quillraven.toilapp.service.ImageService
+import com.github.quillraven.toilapp.service.ToiletService
+import com.github.quillraven.toilapp.service.UserService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.CommandLineRunner
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+import org.springframework.context.ConfigurableApplicationContext
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.ComponentScan.*
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.FilterType
+import org.springframework.data.mongodb.ReactiveMongoDatabaseFactory
+import org.springframework.data.mongodb.ReactiveMongoTransactionManager
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter
+import org.springframework.data.mongodb.core.geo.GeoJsonPoint
+import org.springframework.data.mongodb.gridfs.ReactiveGridFsTemplate
+import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+import java.io.InputStream
+import java.util.*
+import java.util.concurrent.Callable
+import kotlin.random.Random
+import kotlin.system.exitProcess
+
+val titles = listOf(
+    "Beautiful Toilet", "Shit Heaven", "Plumber's Paradies", "Flipi's World", "Diarrhea nightmares", "Decent public toilet", "When you gotta you gott go"
+)
+val descriptions = listOf(
+        "Very nice loo.",
+        "No toilet paper :( :(",
+        "Only on layer toilet paper :(",
+        "Very nice pooping experience.",
+        "One of the most beautiful toilets in the neighborhood.",
+        "Best crap ever!"
+)
+val commentTexts = listOf( "It was a pleasure", "Pleasant shit. I'll come back for sure.")
+val userNames = listOf(
+        "user1",
+        "namibian",
+        "specialist",
+        "homes",
+        "imaginary",
+        "weathertop",
+        "banana",
+        "plink",
+        "manrope,",
+        "rapid",
+        "moonraker",
+        "wholesale",
+        "rundown",
+        "movement",
+        "partridge",
+        "obsessed",
+        "pannikin"
+)
+
+@Configuration
+@EnableAutoConfiguration
+@EnableReactiveMongoRepositories(basePackages = ["com.github.quillraven.toilapp.repository"])
+@ComponentScan(
+        basePackages = ["com.github.quillraven.toilapp.model",  "com.github.quillraven.toilapp.repository",  "com.github.quillraven.toilapp.service",
+            "com.github.quillraven.toilapp_dataloader"]
+)
+class DataLoaderApplication
```

Comment:
`@SpringBootApplication` is the combination of Configuration, EnableAutoConfiguration and ComponentScan. I think we can reduce the annotations to

```kotlin
@SpringBootApplication(
    scanBasePackages = [
        "com.github.quillraven.toilapp.model",
        "com.github.quillraven.toilapp.repository",
        "com.github.quillraven.toilapp.service",
        "com.github.quillraven.toilapp_dataloader"
    ]
)
```